src/analyze/analyze-capability.c

```diff
@@ -20,27 +32,49 @@ int verb_capabilities(int argc, char *argv[], void *userdata) {
         /* Determine the maximum of the last cap known by the kernel and by us */
         last_cap = MAX((unsigned) CAP_LAST_CAP, cap_last_cap());
 
-        if (strv_isempty(strv_skip(argv, 1)))
+        if (arg_capability == ARG_CAP_MASK || (argc == 2 && argv[1][0] == '@')) {
+                uint64_t cap_mask;
+
+                if (argc != 2)
+                        return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "Exactly 1 positional argument expected.");
+
+                if (argv[1][0] == '@')
+                        ++argv[1];
```

Comment:
hmm, i was thinking to support *either* the `@` syntax or the `--mask`/`-m` syntax, not both.

I am not a big fan of redundant interfaces tbh (admittedly `-m` and `--mask` are also kinda redundant, but let's not add a third level of redundancy please..

I think `-m`/`--mask` is the better approach, hence please let's bury the `@` prefix thing, I am sorry for having brought it up ;-)

Reply:
Please drop it, this is not a pattern we use with these tools, so it would be unique and confusing